vagabond/app/views/cities/show.html.erb

```diff
@@ -4,6 +4,6 @@
   <img src="<%= @city.image %>" />
 </div>
 
-<% @city.posts.each do |post|  %>
+<% @city.posts.order("created_at desc").each do |post|  %>
```

Comment:
This is great, but it's considered a no-no to do this kind of complex query in the view. Move it into your controller!